community/examples/Gallery.shtml

```diff
@@ -36,18 +39,15 @@ in the Fall 2016 issue of the
 <a href="http://thamesidersmrg.org.uk/ho-scale-modules-essex-belt-lines/signalling-essex-belt-lines/">OPSIG BR eNews</a>.
 
 <h3>OWNry</h3>
-
-<!--
-<a href="http://ownry.squarespace.com/display/ShowImage?imageUrl=/storage/blog-photos/CTC_Panel_June2006.gif?__SQUARESPACE_CACHEVERSION=1302156072951"/>
-<img src="http://ownry.squarespace.com/storage/thumbnails/5519763-11612079-thumbnail.jpg" align="right" width="200" height="135"/>
-</a>
--->
-<a href="images/5519763-11612079-thumbnail.gif"><img src="images/5519763-11612079-thumbnail.gif" class="floatRight" width="200" height="135" alt="OWN panel" /></a>
-
+<p>
 Kent Williams'
-<a href="http://ownry.squarespace.com">Oregon Washington Navigation &amp; Railway</a>
-(now dismantled) used PanelPro for it's signal controls and
-<a href="http://ownry.squarespace.com/blog/2011/4/7/ctc-dispatcher-panel-on-the-ownry.html?SSScrollPosition=0">dispatcher panel</a>.
+<a href="https://web.archive.org/web/20160818193334/http://ownry.squarespace.com/">Oregon Washington Navigation &amp; Railway</a>
+(now dismantled) used PanelPro for its signal controls and
```

Comment:
The external inks on lines 44 and 47 appear not to work.  Noted in Issue #395.  May not be possible to find...

Reply:
The links open with a delay, but they open on Safari Mac